be/src/service/service_be/starrocks_be.cpp

```diff
@@ -272,6 +272,12 @@ void start_be(const std::vector<StorePath>& paths, bool as_cn) {
     if (config::brpc_num_threads != -1) {
         options.num_threads = config::brpc_num_threads;
     }
+    if (config::enable_https) {
```

Comment:
this is the brpc http/https enablement, not the BE 8040 http server enabling with https. is it what you want?

Reply:
@kevincai yes. this is for brpc http/https. ssl support for BE 8040 http server is work in progress. will raise PR for that later.